.gitignore

```diff
@@ -28,4 +28,4 @@ ledger/
 tests/snapshots-tmp/
 
 # Directory storing the ethereum build output for tests run
-tests/ethereum_build/build/
+tests/.test_dependencies/build/
```

Comment:
This one is not correct, the whole `.test_dependencies` folder should be ignored (since the content depends on the manifest), there won't be a "build" subfolder here anymore.